wa/framework/target/descriptor.py

```diff
@@ -541,6 +541,10 @@ def get_descriptions(self):  # pylint: disable=no-self-use
                           description='''
                           Directory containing Android data
                           '''),
+                Parameter('persistent_data_path', kind=str, default='/storage/emulated/0/Android/data',
+                          description='''
+                          Directory containing persistent Android data
+                          '''),
                ], None),
     'chromeos': ((ChromeOsTarget, 'ChromeOsConnection', []), COMMON_TARGET_PARAMS +
```

Comment:
I assume this should be the same for a ChomeOS target? In which case should we include a similar parameter for that target as well?